src/Symfony/Component/Security/Http/Authenticator/Passport/Passport.php

```diff
@@ -59,6 +58,34 @@ public function getUser(): UserInterface
         return $this->user;
     }
 
+    /**
+     * @return $this
+     */
+    public function addBadge(BadgeInterface $badge): PassportInterface
```

Comment:
```suggestion
    public function addBadge(BadgeInterface $badge): self
```

to be updated to `: static` on 6.0

Reply:
Unfortunately, this breaks PHP <7.4 https://3v4l.org/Nf1MJ

Reply:
Argh, of course